.idea/runConfigurations/cherrypy.xml

```diff
@@ -0,0 +1,30 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="cherrypy" type="PythonConfigurationType" factoryName="Python" singleton="true" show_console_on_std_out="true" show_console_on_std_err="true">
+    <option name="INTERPRETER_OPTIONS" value="" />
+    <option name="PARENT_ENVS" value="true" />
+    <envs>
+      <env name="PYTHONUNBUFFERED" value="1" />
+    </envs>
+    <option name="SDK_HOME" value="ssh://vagrant@127.0.0.1:2222/home/vagrant/magfest/env/bin/python3" />
+    <option name="WORKING_DIRECTORY" value="/home/vagrant/magfest" />
+    <option name="IS_MODULE_SDK" value="false" />
+    <option name="ADD_CONTENT_ROOTS" value="true" />
+    <option name="ADD_SOURCE_ROOTS" value="true" />
+    <module name="magfest" />
+    <EXTENSION ID="PythonCoverageRunConfigurationExtension" enabled="false" sample_coverage="true" runner="coverage.py" />
+    <PathMappingSettings>
+      <option name="pathMappings">
+        <list>
+          <mapping local-root="$PROJECT_DIR$" remote-root="/home/vagrant/magfest" />
+        </list>
+      </option>
+    </PathMappingSettings>
+    <option name="SCRIPT_NAME" value="/home/vagrant/magfest/uber/run_debug_server.py" />
+    <option name="PARAMETERS" value="" />
+    <RunnerSettings RunnerId="PyDebugRunner" />
+    <RunnerSettings RunnerId="PythonRunner" />
+    <ConfigurationWrapper RunnerId="PyDebugRunner" />
+    <ConfigurationWrapper RunnerId="PythonRunner" />
+    <method />
+  </configuration>
+</component>
```

Comment:
ignore, this is OK, I accidentally deleted this file in a previous merge. pycharm stuff.